tf/test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -e
+
+export TF_IN_AUTOMATION=1
+export TF_CLI_ARGS=-no-color
+
+cleanup() {
+  terraform destroy -auto-approve >> tf.log
+}
+
+trap "cleanup" EXIT
+
+echo "-> Creating the underlying infrastructure..."
+terraform init | tee tf.log
+terraform apply -auto-approve | tee -a tf.log
+
+echo "-> Calling the lambda function..."
+aws lambda invoke --function-name smoke-testing-test response.json
+aws lambda invoke --function-name smoke-testing-test response.json
```

Comment:
```suggestion
aws lambda invoke --region=eu-central-1 --function-name smoke-testing-test response.json
aws lambda invoke --region=eu-central-1 --function-name smoke-testing-test response.json
```

If you don't do this, the CLI's configured default region is used.